src/App.vue

```diff
@@ -130,13 +137,22 @@ export default {
         if (!isUserInWorkspace) this.$router.replace({ name: "Home" });
       }
     },
+    $route(to, from) {
+      // show user logged out toast when app is routed to login page
+      // and only when the logout action was not triggered by the user
+      if (to.name == "Login" && from.name != undefined && !this.userClickedLogout) {
+        this.toast.error(this.$t("error.auto_logout"));
+      }
+    },
   },
   methods: {
     // object spread operator
     // https://vuex.vuejs.org/guide/state.html#object-spread-operator
     ...mapActions("auth", ["unsetAccessToken", "fetchAndUpdateUser"]),
     ...mapActions("sync", ["stopLoading"]),
-    logoutUser() {
+    logoutUser(isUserTriggered = false) {
+      // set whether the logout action as triggered by the user or not
+      this.userClickedLogout = isUserTriggered;
```

Comment:
can first have separate function - clickLogout triggered by onclick where this param is set and then this function does not need to accept any param.